boot/kernel_os_test.go

```diff
@@ -289,8 +289,7 @@ func (s *ubootKernelOSSuite) TestExtractKernelAssetsAndRemoveOnUboot(c *C) {
 	c.Assert(err, IsNil)
 
 	// this is where the kernel/initrd is unpacked
-	bootdir := loader.Dir()
-	kernelAssetsDir := filepath.Join(bootdir, "ubuntu-kernel_42.snap")
+	kernelAssetsDir := filepath.Join(dirs.GlobalRootDir, "/boot/uboot/ubuntu-kernel_42.snap")
```

Comment:
nitpick: because of the dirs.GlobalRootDir bit, which we don't use that frequently, I have a slightly preference to stick a bootdir field on the suites

Reply:
Interesting idea, is http://paste.ubuntu.com/p/TxjrGbQpp3/ what you have in mind? Happy to commit it if I understood the suggestion correctly.